deps/eigen/README.md

```diff
@@ -0,0 +1,8 @@
+## Eigen
+
+A clone of only the needed sources (headers) of [Eigen](http://eigen.tuxfamily.org/index.php?title=Main_Page).
```

Comment:
Just so we don't forget: We should discuss with @PeterMatula whether it would be possible to use an external project and download the sources of Eigen when building RetDec so we don't need to have all the 3rd-party sources in our repo. We do this for many of our dependencies.